components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/FtpOperations.java

```diff
@@ -524,7 +524,10 @@ private boolean retrieveFileToFileInLocalWorkDirectory(String name, Exchange exc
             local = new File(local, relativeName);
 
             // create directory to local work file
-            local.mkdirs();
+            boolean result = local.mkdirs();
+            if (!result) {
+                log.error("mkdirs() failed for " + local);
```

Comment:
where can I find example of log markers?

Reply:
By using log markers, I mean using `{}` instead of concatenating the log messages. As such, I am suggesting using something like: 

`log.warn("Failed to create local directory {} while retrieving file in local work directory. Directory may already exist or have been created externally", local);`.

Reply:
Thanks again!